packages/sdk/src/modules/did/index.ts

```diff
@@ -12,20 +12,30 @@ import { CreateStorageKeysEnum } from '../../types';
 import { Base } from '../base';
 
 export interface CustomDocumentFields {
-  services: DocumentService[];
+  verifications?: Verification[],
+  signatures?: Signature[],
```

Comment:
signature shouldn't be an array.
Issuer/Authority signature is a single object.